_blog/why-null-models-crispr.md

```diff
@@ -0,0 +1,83 @@
+
+Many different research groups have observed the same phenomenon: when bacteria use their CRISPR immune system, some spacers end up being a lot more common in the population than others. A *lot* more common: in the data we analyzed for this project, the most common spacer sequence was *ten thousand* times more common than the least common sequence. Distributions like this are often called ''long-tailed'' because in a histogram the data is smeared out for a long way on one or both sides: a long tail.
+
+This figure shows the abundance of each spacer sequence in the experiment, and each colour is a different day in the 14-day experiment. Each sequence is sorted from most common to least common (left to right), and the y-axis shows the frequency of each sequence. For most days, the most common sequence at rank 1 has a frequency of 10<sup>-1</sup> = 10%, and the least common sequences have frequencies around 10<sup>-5</sup>.
+
+<p align="center">
+<img src="/img/posts/why-null-models-crispr/banfield_data.svg" alt="banfield_data" width="400px" style="padding:5px;">
+</p>
+
+This study also found that certain regions of the phage genome were more likely to be a source for spacers, and combined with this skewed abundance distribution, they concluded that the high-abundance spacers were special: their sequence makes them more likely to be high abundance. This sounds like a perfectly reasonable explanation, and it's supported by other experiments. Here’s a figure from a another paper that looked specifically at the process of spacer acquisition - they sequenced new spacers 24 hours after bacteria were infected by phages without leaving the bacteria lots of time to grow and die. They performed the same experiment twice (two 'replicates'), and this plot shows the frequency of spacer sequences in the first replicate versus the frequency of the same spacer sequences in the second replicate. The data points are strongly correlated: they fall on or near a straight diagonal line with slope 1. This means that if a spacer sequence was highly abundant in the first experiment, it is also generally highly abundant in the second experiment, suggesting that there is indeed something about the spacer itself that destines it for high or low abundance[^3]. 
+
+[^3]: This [theory paper](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1005486) included variable spacer effectiveness as an ingredient to get broad abundance distributions.
+
+<p align="center">
+<img src="/img/posts/why-null-models-crispr/Heler2019.svg" alt="Heler2019" width="450px" style="padding:5px;">
+</p>
+
+So we're done, right? We know why some spacers are highly abundant and others are not, and if we see a broad abundance distribution we know the cause. Or do we? Remember where we started: to figure out what's going on when we observe something, we have to rule out other explanations. Could there be other explanations for a spacer abundance distribution that looks like this? 
```

Comment:
Nice! This is very clear here, but wasn’t as much in the intro (see comments there).